src/resources/stepTemplates.ts

```diff
@@ -0,0 +1,48 @@
+import { RouteNames } from '@/enums'
+
+/** The base step template.
+ * Currently used for BEN, BC, ULC and CCC.
+ */
+export const BaseStepsTemplate = [
+  {
+    id: 'step-1-btn',
+    step: 1,
+    icon: 'mdi-domain',
+    text: 'Define Your\nCompany',
+    to: RouteNames.DEFINE_COMPANY,
+    component: RouteNames.DEFINE_COMPANY
```

Comment:
Hmm, this doesn't seem right (that we would have 2 properties with the same value).

I'm thinking "component" maps to the actual component name somehow.

Reply:
I believe it does yea... 🤔  
Just so happens the names match the routes i suppose. Should we create a duplicate enum with a different name?

Reply:
Very good idea.

Reply:
Hmm, sec.

Do we understand where we get the component name from? Is it like declaring components and Vue automatically converts to `<sub-component>` or `<SubComponent>` ? 

I just think this needs to be clear. 🤔 I'm confused.

Reply:
I'll dig into it 👍

Reply:
So yea, our `:component` property which we pass into `<component :is=componentName>` is just like using the component in the template. ie the name of the component. 
For example, the share structure step would work if we passed in `create-share-structure` or `CreateShareStructure` and so on. 
In this case, our routes are aligned with the component names.

Reply:
I suggest using a different enum, to make it clear it's a different thing (even though it's the same values).

Or going back to strings (with a comment on the first one saying what it is, per your comment above).

Reply:
Updated